ee/kafka_client/test/test_client.py

```diff
@@ -0,0 +1,30 @@
+from django.test import TestCase
+
+from ee.kafka_client.client import _KafkaProducer, build_kafka_consumer
+
+
+class KafkaClientTestCase(TestCase):
+    def setUp(self):
+        self.topic = "test_topic"
+        self.payload = {"foo": "bar"}
+
+        self.producer = _KafkaProducer(test=False)
+        self.consumer = build_kafka_consumer(topic=self.topic, auto_offset_reset="earliest", test=False)
```

Comment:
why test=false?

Reply:
because we want to hit the _real_ kafka and not the test kafka interface